### PR TITLE
LPS-112383,LPS-112387 DepotPermissionCheckerWrapper

### PR DESCRIPTION
**Pull request by Preston-Crary**

### CODE REVIEW

**Comment by Preston-Crary** [comment]
ci:test:sf

**Comment by Preston-Crary** [comment]
ci:test:relevant

File: portal-kernel/src/com/liferay/portal/kernel/model/role/RoleConstants.java [hunk 1]
```diff
@@ -102,6 +102,10 @@
 
 	public static final String TYPE_SITE_LABEL = "site";
 
+	public static final int[] TYPES_DEPOT_AND_REGULAR = {
```
Comment (shuyangzhou) [inline portal-kernel/src/com/liferay/portal/kernel/model/role/RoleConstants.java:105]: @adolfopa Why is this not a private constant inside PortletConfigurationPermissionsDisplayContext? What's the point of making it an api constant and causing the package version bump?

Comment (adolfopa) [reply portal-kernel/src/com/liferay/portal/kernel/model/role/RoleConstants.java:105]: I tried to keep all related constants in the same class, that's the only reason. Feel free to move them to a private class.

Thanks!

**Comment by shuyangzhou** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/9045

**Pull request closed without merging**